README.md

```diff
@@ -29,6 +44,7 @@ To build image-inspector using godep:
 
 # Running as a container
 
-    $ docker run -ti --rm --privileged \
+    $ docker run -ti --rm --privileged -p 5000:5000 \
```

Comment:
@enoodle I have the feeling that when this is consumed we use `8080` more often. Anyway it's a nit/taste.